e2e/cloud_manager/dbusers_test.go

```diff
@@ -72,16 +73,23 @@ func TestDBUsers(t *testing.T) {
 			"--password=passW0rd",
 			"--role=readWriteAnyDatabase",
 			"--mechanisms=SCRAM-SHA-256")
-		cmd.Env = os.Environ()
-		resp, err := cmd.CombinedOutput()
 
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
-		}
+		testCreatePasswordCmd(t, cmd)
+	})
 
-		if !strings.Contains(string(resp), "Changes are being applied") {
-			t.Errorf("got=%#v\nwant=%#v\n", string(resp), "Changes are being applied")
-		}
+	t.Run("CreatePasswordStdin", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			entity,
+			dbUsersEntity,
+			"create",
+			"--username="+username,
+			"--role=readWriteAnyDatabase",
+			"--mechanisms=SCRAM-SHA-256")
+
+		passwordStdin := bytes.NewBuffer([]byte("passW0rd"))
+		cmd.Stdin = passwordStdin
+
+		testCreatePasswordCmd(t, cmd)
```

Comment:
[2021/11/15 15:28:14.823] FAILURE: Failed ()
 [2021/11/15 15:28:14.823] === RUN   TestDBUsers/CreatePasswordStdin
 [2021/11/15 15:28:14.823]     dbusers_test.go:92: unexpected error: exit status 1, resp: Error: PUT https://cloud-dev.mongodb.com/api/public/v1.0/groups/61927b10ab680d0601bf033a/automationConfig: 400 (request "") Invalid config: Each auth.usersWanted must have a unique user id. Duplicate found: user-126@admin
 [2021/11/15 15:28:14.823]     --- FAIL: TestDBUsers/CreatePasswordStdin (0.15s)

Reply:
ah, same stuff as the atlast tests, will take a look at pulling these out into `TestDBUsersWithFlags` and `TestDBUsersWithStdin` too.